etcdmain/etcd.go

```diff
@@ -605,3 +604,15 @@ func setupLogging(cfg *config) {
 		repoLog.SetLogLevel(settings)
 	}
 }
+
+func checkSupport() {
+	// TODO qualify arm64
+	if runtime.GOARCH == "amd64" {
+		return
+	}
+	if env, ok := os.LookupEnv("ETCD_UNSUPPORTED_ARCH"); ok && env == runtime.GOARCH {
+		return
```

Comment:
print out warning here? like etcd is running with unsupported arch X since unsupported_arch env is set?